lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -1222,7 +1222,7 @@ protected function doModifyLimitQuery($query, $limit, $offset = null)
      */
     private function scrubInnerOrderBy($query)
     {
-        $count = substr_count(strtoupper($query), "ORDER BY");
+        $count = substr_count(strtoupper($query), " ORDER BY");
```

Comment:
This patch doesn't help if you put a whitespace in front of `ORDER BY` in your query.
Then this clause`OVER( ORDER BY t0_.test2 DESC)` will get converted to this `OVER()` :(

I guess we are hitting limits here without a real SQL parser...